std/uuid.d

```diff
@@ -148,6 +148,12 @@ public struct UUID
                 179, 189, 251, 70])._toString() == "8ab3060e-2cba-4f23-b74c-b52db3bdfb46");
         }
 
+        // Reinterpret the UUID as an array of some other primitive.
+        @trusted ref T[16 / T.sizeof] asArrayOf(T)()
```

Comment:
So now, `@` is optional on every keyword. Is there a new manual of style to say which I should prefer (`trusted` vs `@trusted` or `return` vs `@return`)?